docker-compose.yml

```diff
@@ -3,8 +3,11 @@ services:
     container_name: sotf-server
     image: chemineer/sons-of-the-forest-dedicated:latest
     #build: .
-    network_mode: "host"
-    restart: always
     init: true
+    ports:
+      - "8766:8766/udp" # Game port
+      - "27016:27016/udp" # Query port
+      - "9700:9700/udp" # Blob sync port
     volumes:
       - ./userdata:/app/sonsoftheforest/userdata
+    restart: always
```

Comment:
Is there a reason for moving `restart: always` to this line?

Reply:
No